core/src/main/kotlin/org/evomaster/core/output/oracles/SupportedCodeOracle.kt

```diff
@@ -47,27 +48,35 @@ class SupportedCodeOracle : ImplementedOracle() {
             val supportedCodes = getSupportedCode(call)
             //BMR: this will be a problem if supportedCode contains both codes and default...
             if(supportedCodes.contains("0")){
-                lines.add("// WARNING: the code list seems to contain an unsupported code (0 is not a valid HTTP code). This could indicate a problem with the schema. The issue has been logged.")
+                lines.addSingleCommentLine("WARNING: the code list seems to contain an unsupported code (0 is not a valid HTTP code). This could indicate a problem with the schema. The issue has been logged.")
                 supportedCodes.remove("0")
                 LoggingUtil.uniqueWarn(log, "The list of supported codes appears to contain an unsupported code (0 is not a valid HTTP code). This could indicate a problem with the schema.")
                 //TODO: if a need arises for more involved checks, refactor this
             }
             val supportedCode = supportedCodes.joinToString(", ")
 
             if(supportedCode.equals("")){
-                lines.add("/*")
-                lines.add(" Note: No supported codes appear to be defined. https://swagger.io/docs/specification/describing-responses/.")
-                lines.add(" This is somewhat unexpected, so the code below is likely to lead to a failed expectation")
-                lines.add("*/")
+                if (format.isJavaOrKotlin()) {
+                    addNoSupportedCodeComments(lines, "below")
+                }
                 when {
                     format.isJava() -> lines.add(".that($variableName, Arrays.asList().contains($name.extract().statusCode()))")
                     format.isKotlin() -> lines.add(".that($variableName, listOf<Int>().contains($name.extract().statusCode()))")
+                    format.isPython() -> lines.add(".that(self.$variableName, $name.status_code in [])")
```

Comment:
@arcuri82 I've got an issue here. Python does not support having a comment in between method calls, something that in java would look like:

anObject.doSomething()
// a comment
.doSomethingElse()

My first thought was not to use the fluent API and thus have every `that` call in a separate method, having for example:
`expectation_handler.expect(ems).that(swtich1, condition1)`
`expectation_handler.expect(ems).that(swtich2, condition2)`

The problem I bumped into is that the expectationsMasterSwitch is a private value in the [PartialOracles](https://github.com/WebFuzzing/EvoMaster/pull/1023/files#diff-332fc827e7d77b6da33b13013fff415eb36c504b36765a6976a36657782c791dR86) class. Do you have any suggestions?

Reply:
@Pgarrett  hi. don't need to worry about this one... "oracles" and "expectations" are going to be removed due to a planned refactoring related to the reporting of detected faults (which will be done in the next future).
so, for time being, can just skip it (and leave a comment)